### PR TITLE
cdc,regionspan(ticdc): more logs about eventfeed retry rate limit

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Add more logs to facilitate debug for #4006 

### Check List 

Tests 

 - Unit test

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/kv/client.go [hunk 1]
```diff
@@ -520,7 +520,12 @@ func (s *eventFeedSession) eventFeed(ctx context.Context, ts uint64) error {
 				s.errChSizeGauge.Dec()
 				allowed := s.checkRateLimit(errInfo.singleRegionInfo.verID.GetID())
 				if !allowed {
-					// rate limit triggers, add the error info to the rate limit queue
+					// rate limit triggers, add the error info to the rate limit queue.
+					log.Info("EventFeed retry rate limited",
+						zap.Uint64("regionID", errInfo.singleRegionInfo.verID.GetID()),
+						zap.Uint64("ts", errInfo.singleRegionInfo.ts),
+						zap.Int64("tableID", errInfo.singleRegionInfo.span.TableID()),
```
Comment (amyangfei) [inline cdc/kv/client.go:527]: Can log `span` info, besides table id has been stored in context, which can be used directly.

**Review by amyangfei — APPROVED** [review]

**Review by ben1009 — APPROVED** [review]

**Pull request merged**